.github/workflows/artifacts.yaml

```diff
@@ -23,35 +23,34 @@ jobs:
         password: ${{ secrets.GITHUB_TOKEN }}
       env:
         NODE_HW: CPUS=2
+        SLURM_VER: ${{ matrix.version }}
       options: --cpus 2 --privileged
 
     steps:
       - uses: actions/checkout@v3
       - run: |
-          make rpm
+          make RPM_SLURM_VERSION=${SLURM_VER} rpm
           sudo rpmbuild --rebuild "$(find rpm/SRPMS -name '*.rpm' -type f -print -quit)" --define "_topdir $(pwd)/rpm"
-          sudo rpm --install "$(find rpm/RPMS -name '*.rpm' -type f -print -quit)"
+          # TODO: this fails, because the new rpm lists slurm as dependency (image installed slurm manually not via pkg manager)
```

Comment:
Is this still the case?
If it is, maybe don't mark SLURM as required in the RPM.

Reply:
Is SLURM installed as an RPM on the Alps clusters?

Reply:
Yes, on ALPS slurm is installed as RPM. But not in the container.

Reply:
I've added `rpm --install --nodeps` in the line above. Or shall we remove the explicit dependency on slurm from the rpm package?

Reply:
I added slurm as a dependency after we had the issue that the path for the plugin conf file did not exist if this package is installed before slurm. The dependency forces the installation order to be correct. Otherwise we have to explicitly create the directory when the rpm is installed, or address the issue in some other way.